jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -2161,12 +2161,15 @@ public String toGenericString() {
  * Returns the Package of which this class is a member.
  * A class has a Package iff it was loaded from a SecureClassLoader
  *
- * @return		Package the Package of which this class is a member or null
+ * @return		Package the Package of which this class is a member or null in the case of primitive or array types.
```

Comment:
Please avoid long lines.

Reply:
Line deliberately not wrapped per instructions from @DanHeidinga .